README.md

```diff
@@ -251,7 +263,6 @@ GET     /file-transfer/envelopes/{envelope-id}
 | Ok  | 200   | File Successfully uploaded.  |
```

Comment:
This section is all wrong... Where did you get 'file successfully uploaded' when this is download end-point? Where did you get 206 partially downloaded? Are we returning sth like that?